packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/views/ViewManagerDefinitionBuilder.kt

```diff
@@ -66,6 +66,52 @@ class ViewManagerDefinitionBuilder {
     )
   }
 
+  /**
+   * Creates a view prop group that defines its name and setter.
+   */
+  inline fun <reified ViewType : View, reified PropType> Prop(
+    vararg names: String,
+    noinline body: (name: String, view: ViewType, prop: PropType) -> Unit
```

Comment:
I don't have a strong opinion, but imho the view should always be the first 🤔 and maybe the prop argument be the second to keep the order the same as the basic `Prop`. What do you think?

Reply:
I can agree with the view being the first, but I think the `name`/`index` should be in second place.